ga4gh/datarepo.py

```diff
@@ -193,3 +197,10 @@ def __init__(self, dataDir):
                 relativePath = os.path.join(sourceDir, setName)
                 if os.path.isdir(relativePath):
                     objectAdder(constructor(setName, relativePath, self))
+        if doConsistencyCheck:
+            self._checkConsistency()
+
+    def _checkConsistency(self):
```

Comment:
This will be fixed in a change that is about to be merged, but all checkConsistency methods should be public, not private (that is, get rid of the underscore)